config/lists/categories.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" ?>
+<list xmlns="http://schemas.sulu.io/list-builder/list">
+    <key>categories</key>
```

Comment:
Sulu automatically merges list-configurations that use the same key. This allows to hide existing properties and display new properties in existing lists.